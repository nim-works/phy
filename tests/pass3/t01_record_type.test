(TypeDefs
  (Int 4)
  (Record 16 (Field 0 (Type 0)) (Field 0 (Type 0)))
  (ProcTy (Void))
)
(GlobalDefs)
(ProcDefs
  (ProcDef (Type 2) (Locals) (Continuations
    (Continuation (Params)))
  )
)