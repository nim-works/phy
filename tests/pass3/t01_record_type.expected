;$sexp
(TypeDefs
  (Int 4)
  (Blob 16)
  (ProcTy (Void))
  (UInt 8)
)
(GlobalDefs)
(ProcDefs
  (ProcDef (Type 2) (Locals)
    (Continuations
      (Continuation (Params))
    )
  )
)
